front/app/containers/SearchPage/SearchView.tsx

```diff
@@ -727,7 +727,6 @@ class SearchView extends React.Component<SearchViewProps, SearchViewState> {
 
   renderViewDropdown = () => {
     const { currentSiteView } = this.props;
-    console.log(currentSiteView.search.results.buttons);
```

Comment:
we really need to prevent `console.log` statements from getting merged in during CRs. we could turn the linter into an error for it in CI if we had CI.